src/Altinn.Apps/AppTemplates/AspNet/App.IntegrationTests/Data/apps/dibk/nabovarsel/config/applicationmetadata.json

```diff
@@ -0,0 +1,41 @@
+{
+  "id": "dibk/nabovarsel",
+  "org": "dibk",
+  "created": "2019-09-24T10:02:41.0839253Z",
+  "createdBy": "Kritsi",
+  "lastChanged": "2019-09-24T10:02:41.0839254Z",
+  "lastChangedBy": "Kritsi",
+  "title": {
+    "nb": "Nabovarsel)",
+    "nb-NO": "Nabovarsel"
+  },
+  "dataTypes": [
+    {
+      "id": "melding",
+      "allowedContentTypes": [ "application/xml" ],
+      "maxCount": 1,
+      "appLogic": {
+        "autoCreate": false,
+        "ClassRef": "App.IntegrationTestsRef.Data.apps.dibk.nabovarsel.Melding"
+      },
+      "taskId": "Task_1"
+    },
+    {
+      "id": "skjema",
+      "allowedContentTypes": [ "application/xml" ],
+      "maxCount": 1,
+      "appLogic": {
+        "autoCreate": false,
+        "ClassRef": "App.IntegrationTestsRef.Data.apps.dibk.nabovarsel.SvarPaaNabovarselType"
+      },
+      "taskId": "Task_2"
+    },
+    {
+      "id": "9edd53de-f46f-40a1-bb4d-3efb93dc113d",
```

Comment:
this is the attachment from the app owner? I'm guessing we would expect the name to not be a guid as this property will have to be created manually by the app owner?

Reply:
Agree. Renamed to something more functional